packages/manager/modules/pci/src/projects/project/storages/cold-archive/add/add.routing.js

```diff
@@ -17,6 +20,37 @@ export default /* @ngInject */ ($stateProvider) => {
           'pci_projects_project_storages_cold_archive_add_breadcrumb',
         ),
 
+      guideMenu: /* @ngInject */ (initGuides) => initGuides(GUIDE_MENU_ITEMS),
+
+      initGuides: /* @ngInject */ (coreConfig, $translate) => (guides) => {
+        return guides.reduce(
+          (list, guide) => [
+            ...list,
+            {
+              ...guide,
+              title: $translate.instant(
+                `pci_projects_project_storages_cold_archives_guides_${guide.id}_title`,
+              ),
+              description: $translate.instant(
+                `pci_projects_project_storages_cold_archives_guides_${guide.id}_description`,
+              ),
+              link:
+                guide.links[coreConfig.getUser()?.ovhSubsidiary] ||
+                guide.links.DEFAULT,
+            },
+          ],
+          [],
+        );
+      },
```

Comment:
**nitpick:** Since it is not used template, this can be combined into `guidemenu` and avoid unnecessary injection.

Reply:
Done